astropy/io/votable/tests/table_test.py

```diff
@@ -181,6 +181,24 @@ def test_empty_table():
     astropy_table = table.to_table()  # noqa
 
 
+def test_binary2_masked_strings():
+    """
+    Issue #8995
+    """
+    # Read a VOTable which sets the null mask bit for each empty string value.
+    votable = parse(get_pkg_data_filename('data/binary2_masked_strings.xml'))
+    table = votable.get_first_table()
+    astropy_table = table.to_table()
+
+    # Ensure string columns have no masked values and can be written out
+    assert not np.any(table.array.mask['epoch_photometry_url'])
+    output = io.BytesIO()
+    astropy_table.write(output, format='votable')
+
+    # TODO - Construct test cases for unicodeChar strings and fixed length
```

Comment:
Do you have plans to follow up on this as part of this PR?

Reply:
I thought more about this and realize that the TODO is a larger homework assignment for me to better review/understand the existing test coverage.  That is outside the scope of this PR, so I filed issue #9433 and removed the TODO comment.